crt_portal/cts_forms/templates/forms/pro_template.html

```diff
@@ -23,16 +23,17 @@ <h2 id="intake">Intake</h2>
   </div>
 </div>
 
-<div class="crt-portal-card">
+<div class="crt-portal-card crt-portal-card--proform">
   <div class="crt-portal-card__content crt-portal-card__content--lg">
-    <a name="contact"></a>
     {% include "forms/question_cards/contact.html" with card_title='Contact' %}
   </div>
 </div>
 
-{% include "forms/question_cards/single_form.html" with field=form.servicemember card_title="Service member" %}
+<div class="crt-portal-card--proform">
```

Comment:
Had to wrap those extra classes around the Service Member and Incident Location cards, since the markup for those was entirely contained in their components.